server/src/test/java/io/crate/metadata/ColumnIdentTest.java

```diff
@@ -114,6 +111,7 @@ public void testIllegalColumnNameValidation() throws Exception {
         assertExceptionIsThrownOnValidation("ident['index']", "subscript");
         assertExceptionIsThrownOnValidation("ident['index]", "subscript");
         assertExceptionIsThrownOnValidation("ident[0]", "subscript");
+        assertExceptionIsThrownOnValidation("\"a[1]\"", "subscript");
```

Comment:
Maybe add another test with a more "random" string like: `adffdafda['foo']`

Reply:
`adffdafda['foo']` seems like a duplicate of `ident['index']`. added `"fda_32$@%^nf[ffDA&^"`, also added `[` to better demonstrate that `[` is forbidden.

Reply:
> adffdafda['foo'] seems like a duplicate of ident['index']
Indeed, thx!